apps/website/src/server/trpc/trpc.ts

```diff
@@ -14,10 +14,24 @@ const t = initTRPC.context<Context>().create({
 
 export const router = t.router;
 
+// Middleware to log mutations
+const logMutations = t.middleware(async ({ path, type, next, ctx }) => {
+  if (type === "mutation") {
+    const userId = ctx.session?.user?.id || "Unauthenticated";
```

Comment:
```suggestion
    const user = ctx.session?.user ? `@${ctx.session.user.name ?? 'unknown'} (${ctx.session.user.id})` : "Unauthenticated";
```
Can we include the user's name here to make it easier to follow the logs at a glance?

Reply:
Applied